src/main/java/com/jhuguet/sb_taskv1/app/models/User.java

```diff
@@ -29,10 +30,13 @@ public class User {
     private int id;
     private String username;
     private String email;
+    @Column(unique = true)
```

Comment:
Why is password unique? There can be case when multiple users use 1 password accidentally.

Reply:
Changed to username, misplaced tag accidentally